logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -16,6 +16,17 @@
 
 import math
 import json
+import sys
+
+try:
+    import flask
+except ImportError:
+    flask = None
+
```

Comment:
Imports at top-level in source code, unless absolutely necessary (e.g. import cycles)